### PR TITLE
add Venidium testnet and mainnet networks

### PR DESCRIPTION
**Pull request by itsfridaythen**
adds 2 more chains
Venidium Testnet: https://chainlist.org/chain/4918
Venidium Mainnet: https://chainlist.org/chain/4919

### CODE REVIEW

**Comment by itsfridaythen** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by itsfridaythen** [comment]
@katspaugh Thanks for the notice.
Even if all services (txs, cfg and cgw) are configured correctly, the UI does not let you select networks without an entry in the chains dictionary.

**Review by katspaugh — APPROVED** [review]

**Review by katspaugh — CHANGES_REQUESTED** [review]
Wait, both chainIds are the same

File: src/config/chains.ts [hunk 1]
```diff
@@ -192,6 +192,8 @@ const chains: Record<string, string> = {
   tftm: '4002',
   'iotex-mainnet': '4689',
   'iotex-testnet': '4690',
+  txvm: '4919',
```
Comment (katspaugh) [inline src/config/chains.ts:195]:
```suggestion
  txvm: '4918',
```

**Review by katspaugh — APPROVED** [review]
👍

**Pull request merged**